sql/2022/compression/compression_by_content_type.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# compression_by_content_type.sql : Compressopnn methods for different content types
+SELECT
+  client,
+  mimeType,
+  resp_content_encoding,
+  COUNT(0) AS num_requests,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
+FROM
+  `httparchive.almanac.requests`
+WHERE
+  date = '2022-06-01'
+GROUP BY
+  client,
+  mimeType,
+  resp_content_encoding
+HAVING
+  num_requests > 1000
+ORDER BY
+  num_requests DESC
```

Comment:
This having will affect the totals since it's applied before windowing clause. Suggest you change to `QUALIFY`

```suggestion
ORDER BY
  num_requests DESC
QUALIFY
  num_requests > 1000
```